### PR TITLE
Add support for Entry Point commands

### PR DESCRIPTION
**Pull request by mkrasnitski**
The discord docs are a little confusing for this feature, but I think it's basically pretty simple. Applications with activities enabled can define a global command of type `PrimaryEntryPoint` - based on the value of the `handler` field, this command can be treated essentially like normal, allowing the application to respond with a followup message when called; or, the behavior of the command can be automatically handled by Discord.

### CODE REVIEW

File: src/builder/create_interaction_response.rs [hunk 1]
```diff
@@ -64,6 +64,13 @@ pub enum CreateInteractionResponse {
     /// Corresponds to Discord's `PREMIUM_REQUIRED'.
     #[deprecated = "use premium button components via `CreateButton::new_premium` instead"]
     PremiumRequired,
+    /// Not valid for autocomplete and Ping interactions. Only available for applications with
+    /// Activities enabled.
+    ///
+    /// Responds to the interaction by launching the Activity associated with the app.
```
Comment (GnomedDev) [inline src/builder/create_interaction_response.rs:70]: Can these lines be flipped? Seems to be backwards.

**Review by GnomedDev — COMMENTED** [review]
Small nitpick, but otherwise good to merge.

File: src/builder/create_interaction_response.rs (hunk 1 above)
Comment (mkrasnitski) [reply src/builder/create_interaction_response.rs:70]: This mirrors the docs for the other variants, like `Acknowedge`, `UpdateMessage`, `Autocomplete`, and `Modal`. If this should be changed, I think the others should too.

Comment (GnomedDev) [reply src/builder/create_interaction_response.rs:70]: Ah, no worries then.

**Review by GnomedDev — APPROVED** [review]

**Pull request merged**